docs/application/vstools/tools/manifest-editor.md

```diff
@@ -84,15 +90,16 @@ For more information on the relationship between the elements, see the [element
 **Expected children (in the following order):**
 
 | Child element           | Occurrences          |
-| ----------------------- | -------------------- |
-| `<author>`              | 1 (optional)         |
-| `<description>`         | 1 or more (optional) |
-| `<profile>`             |                      |
-| `<ui-application>`      | 1 (optional)         |
-| `<shortcut-list>`       |                      |
-| `<account>`             |                      |
-| `<privileges>`          |                      |
-| `<feature>`             | 1 or more (optional) |
+|-------------------------|----------------------|
+| [`<author>`](#author)              | 1 (optional)         |
```

Comment:
To keep the consistency of the document, I think we'd better remove links in this chart.

Reply:
Done